geode-core/src/main/java/org/apache/geode/internal/cache/ClusterConfigurationLoader.java

```diff
@@ -31,7 +31,7 @@
 import org.apache.geode.internal.admin.remote.DistributionLocatorId;
 import org.apache.geode.internal.i18n.LocalizedStrings;
 import org.apache.geode.internal.logging.LogService;
-import org.apache.geode.internal.process.ClusterConfigurationNotAvailableException;
+import org.apache.geode.config.internal.ClusterConfigurationNotAvailableException;
```

Comment:
Classic Pat nitpick: don't forget to reorder / optimize your imports to go `statics`, `java`, `javax`, (other), `org.apache.geode`.

This applies to the following files:
```
geode-core/src/main/java/org/apache/geode/distributed/ServerLauncher.java
geode-core/src/main/java/org/apache/geode/distributed/internal/InternalLocator.java
geode-core/src/main/java/org/apache/geode/internal/cache/ClusterConfigurationLoader.java
geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java
geode-core/src/main/java/org/apache/geode/internal/process/LocalProcessLauncher.java
geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/StartServerCommand.java
```